### PR TITLE
 Add GA and Adobe Analytics

### LINKED ISSUE #180: Implement en/fr Google Analytics code
With two domains, we have two GA properties.

EN:
UA-102484926-19

```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-102484926-19"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-102484926-19');
</script>
```

FR:
UA-102484926-20

```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-102484926-20"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-102484926-20');
</script>
```

### PR DESCRIPTION
**Pull request by dsamojlenko**
Closes #180 

- Adds english and french Google Analytics code (production)
- Adds Adobe Analytics code (production)
- Adds required meta information for Adobe
- Removes old starter_repo ga implementation, since we needed to support separate en/fr codes
- Moves some script tags to inside the body tag
- Fixes a test that was expecting the starter_repo ga code

### CODE REVIEW

File: views/base.njk [hunk 1]
```diff
@@ -53,4 +77,5 @@
             {% endfor %}
         {% endif %}
     {% endblock %}
+
```
Comment (pcraig3) [inline views/base.njk:80]: Wait, there shouldn't be stuff outside the body tag?!

Comment (dsamojlenko) [reply views/base.njk:65]: yeah, not sure why that stuff is out there.... will move in.

**Comment by CalvinRodo** [comment]
Comment From principle publisher for form pages:

For the service metadata, I recommend something along the lines of ESDC-EDSC_COVID-BENEFITS, or you can use the name of the tool or abbreviation in place of COVID-BENEFITS. This value can be used to report on the specific application, that’s its only real purpose.

 

Here is the value we’re recommending for the formname (open to changing if the name of the tool changes): data-gc-analytics-formname="ESDC|EDSC:Money for COVID19"

 

Here is the text to capture what radio button is selected: data-gc-analytics-collect="[{"value": "input","emptyField": "n/a"}]"

**Comment by CalvinRodo** [comment]
Allright TODO
- [ ] Don't render __satelite at end of page if Adobe Analytics is not implemented
- [ ] Fix CSRF issue with Adobe Analytics javascript.

**Pull request merged**